_posts/2017-12-20-the-real-magic-of-bridge-foundry.md

```diff
@@ -0,0 +1,32 @@
+tags: ready
+---
+*Post by Mary Jenn*
+
+I’ve been associated consistently with Bridge Foundry (formerly known as RailsBridge) since at least 2010. I started as a student, then became a volunteer teaching assistant, then I started organizing events. As the organization grew and new “bridges” were added and workshops were being held world wide, I began mentoring new organizers and then once Bridge Foundry got off the ground, I became their first paid employee. 
+
+I often find myself clarifying just what Bridge Foundry is about to people. Firstly: While we are about enhancing the diversity of the software engineering community, we are an INCLUSIVE community and we do allow all walks of life into our workshops.  The reason why: we WELCOME the people who typically dominate the industry to be in attendance at an event that is led by people who are not usually in the majority. And to see that those underrepresented people are sharp, capable and intelligent too. 
+
+But here’s the thing: -- hold onto your hats: It’s NOT just about teaching people to code. NOPE. It’s not!  
+
+Wait. What??!!
+
+It’s about so much more than that. 
+
+It’s about transformation, community and the magic that can happen when people who had no idea they could do something that “only ***smart*** ***nerdy*** ***techy*** people” can do. And at our workshops they are empowered to do just that. 
+
+YES! You too, can be a nerd. 
+
+But HERE is where I really get really excited about Bridge Foundry:  The work I’ve done with the organization  has really been a journey of personal growth. I’ve come to get more comfortable with my own strengths and weaknesses. Ive faced my own biases -- we all have them. I’ve learned that we will ALWAYS encounter people I don’t like and don’t really want to work with. But it’s up to me to work issues out if I want to achieve a goal within an organization. 
+
+I’ve learned to be comfortable in an organization full of strong women and people of color — people like me and people who are different from me. And if I want the freedom to use my strong voice, I need to give other people the space to use theirs -- and to be who they are. 
+
+RailsBridge co-founder, Sarah Allen, has said about workshop attendees,  “If anyone has a problem with someone who is not hurting anyone, then THEY need to deal with their discomfort within their own self”. Everyone has something to contribute, you see. Everyone has worth. 
+
+So, This. THIS, my friends is the REAL magic of Bridge Foundry and is at the heart of who we are. This idea really plays into the long view of who we are and where we’re headed.
+
+In the meantime, please share your experiences or thoughts! Have you overcome a particular issue in your community? Have you had your own journey of personal growth? Are you shocked to hear this--that we’re not REALLY just about teaching programming? I’d love to hear in the comments. Blog posts about your own experiences are welcome too and you can post here <link>. 
```

Comment:
@audiodude added instructions in the README about how to make a post, we could say something like:

...you can submit draft posts as a [github issue](https://github.com/bridgefoundry/bridgefoundry.github.io/issues/new) or as a [pull request](https://github.com/bridgefoundry/bridgefoundry.github.io)